diesel/src/sqlite/types/date_and_time/chrono.rs

```diff
@@ -7,21 +8,89 @@ use crate::serialize::{self, IsNull, Output, ToSql};
 use crate::sql_types::{Date, Time, Timestamp, TimestamptzSqlite};
 use crate::sqlite::Sqlite;
 
-const SQLITE_DATE_FORMAT: &str = "%F";
+// Warning to future editors:
+// Changes in the following formats need to be kept in sync
+// with the formats of the "time" module.
+// We do not need a distinction between whole second and
+// subsecond since %.f will only print the dot if needed.
+// We always print as many subsecond as his given to us,
+// this means the subsecond part can be 3, 6 or 9 digits.
```

Comment:
```suggestion
/// Warning to future editors:
/// Changes in the following formats need to be kept in sync
/// with the formats of the ["time"](super::time) module.
/// We do not need a distinction between whole second and
/// subsecond since %.f will only print the dot if needed.
/// We always print as many subsecond as his given to us,
/// this means the subsecond part can be 3, 6 or 9 digits.
```

[as suggested here: ](https://github.com/diesel-rs/diesel/pull/3694/files#r1275723428)